test/e2e/operator.go

```diff
@@ -273,3 +273,52 @@ var _ = Describe("ARO Operator - Conditions", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 })
+
+var _ = Describe("ARO Operator - MachineSet Controller", func() {
+	Specify("operator should maintain at least two worker replicas", func() {
+		// TODO: MSFT Billing expects that we only scale a single node (4 VMs).
+		// Need to work with billing pipeline to ensure we can run operator tests
+		if !_env.IsLocalDevelopmentMode() {
```

Comment:
Instead of writing this if statement, there already exists a function which skips if not localDevelopmentMode
```go
skipIfNotInDevelopmentEnv()
```

Reply:
Done, thanks!